README.md

```diff
@@ -39,13 +35,13 @@ module.exports = {
   plugins: [
     new BundleTracker({
       path: __dirname,
-      filename: './assets/webpack-stats.json',
+      filename: 'assets/webpack-stats.json',
```

Comment:
perhaps we should always reject dirs in filename?
for the path to work here, we can do `path.join(__dirname, 'assets')`